common/lib/dependabot/shared_helpers.rb

```diff
@@ -26,6 +26,19 @@ module SharedHelpers
     def self.in_a_temporary_repo_directory(directory = "/",
                                            repo_contents_path = nil,
                                            &block)
+      if repo_contents_path && Dependabot::Experiments.enabled?(:shared_workspace)
+        stdout1, _proc1 = Open3.capture2("df -k /tmp 2>/dev/null || true")
+        stdout2, _proc2 = Open3.capture2("df -k /home/dependabot 2>/dev/null || true")
+        stdout3, _proc3 = Open3.capture2("df -k /home/dependabot/dependabot-core/tmp 2>/dev/null || true")
+        stdout4, _proc4 = Open3.capture2("df -k /home/dependabot/dependabot-updater/tmp 2>/dev/null || true")
+
+        Dependabot.logger.info("shared_workspace: repo_contents_path: #{repo_contents_path}")
+        Dependabot.logger.info("shared_workspace: df -k /tmp: #{stdout1}")
+        Dependabot.logger.info("shared_workspace: df -k /home/dependabot: #{stdout2}")
+        Dependabot.logger.info("shared_workspace: df -k /home/dependabot/dependabot-core/tmp: #{stdout3}")
+        Dependabot.logger.info("shared_workspace: df -k /home/dependabot/dependabot-updater/tmp: #{stdout4}")
```

Comment:
💅 Fully aesthetic suggestion: What do you think about naming the output variables after the directory they're checking?

```suggestion
        root_tmp, _proc1 = Open3.capture2("df -k /tmp 2>/dev/null || true")
        home, _proc2 = Open3.capture2("df -k /home/dependabot 2>/dev/null || true")
        core_tmp, _proc3 = Open3.capture2("df -k /home/dependabot/dependabot-core/tmp 2>/dev/null || true")
        updater_tmp, _proc4 = Open3.capture2("df -k /home/dependabot/dependabot-updater/tmp 2>/dev/null || true")

        Dependabot.logger.info("shared_workspace: repo_contents_path: #{repo_contents_path}")
        Dependabot.logger.info("shared_workspace: df -k /tmp: #{root_tmp}")
        Dependabot.logger.info("shared_workspace: df -k /home/dependabot: #{home}")
        Dependabot.logger.info("shared_workspace: df -k /home/dependabot/dependabot-core/tmp: #{core_tmp}")
        Dependabot.logger.info("shared_workspace: df -k /home/dependabot/dependabot-updater/tmp: #{updater_tmp}")
```